src/lib/services/data-converter.ts

```diff
@@ -11,9 +12,9 @@ interface WebSocketResponse {
 
 export async function convertPayload(
   payload: Payload,
-  websocket: any,
+  websocket: WebSocketAsPromised,
 ): Promise<string | Payload> {
-  if (!websocket.isOpened && !websocket.isOpening) {
+  if (!websocket.isOpened) {
```

Comment:
If we're opening but not opened we would send before the websocket was opened and send would be broken.